job-service/src/main/java/com/hpe/caf/services/job/api/DatabaseHelper.java

```diff
@@ -78,34 +75,8 @@ public Job[] getJobs(final String partitionId, String jobIdStartsWith, String st
                 final Connection conn = DatabaseConnectionProvider.getConnection(appConfig);
                 final CallableStatement stmt = conn.prepareCall("{call get_jobs(?,?,?,?,?,?,?,?,?,?)}")
         ) {
-            if (jobIdStartsWith == null) {
-                jobIdStartsWith = "";
-            }
-            if (statusType == null) {
-                statusType = "";
-            }
-            if (limit == null) {
-                limit = 0;
-            }
-            if (offset == null) {
-                offset = 0;
-            }
-            stmt.setString(1, partitionId);
-            stmt.setString(2, jobIdStartsWith);
-            stmt.setString(3, statusType);
-            stmt.setInt(4, limit);
-            stmt.setInt(5, offset);
-            stmt.setString(6, sortField.getDbField());
-            stmt.setString(7, sortField.getSortLabel());
-            stmt.setBoolean(8, sortDirection.getDbValue());
-            Array array;
-            if (labels != null) {
-                array = conn.createArrayOf("VARCHAR", labels.toArray());
-            } else {
-                array = conn.createArrayOf("VARCHAR", new String[0]);
-            }
-            stmt.setArray(9, array);
-            stmt.setString(10, filter);
+            buildGetJobsStatement(partitionId, jobIdStartsWith, statusType, limit, offset, sortField,
```

Comment:
DatabaseHelper.java::buildGetJobsStatement() is also in [the bulk cancellation PR](https://github.com/JobService/job-service/pull/362) which should be merged before this PR.